spring-integration-core/src/main/java/org/springframework/integration/graph/IntegrationGraphServer.java

```diff
@@ -132,6 +133,7 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
 	 * @return the graph.
 	 * @see #rebuild()
 	 */
+	@SuppressWarnings("NullAway") // Dataflow analysis limitation
```

Comment:
We need to extract a local variable instead.
Remember all those similar `..ToRemove`, `..ToUse`, `..ToCancel` etc.?